### ISSUE TITLE
Example of a Good Issue / Story / Bug

### ISSUE THREAD

**Issue by nelsonic** [dwyl/contributing#8]
We need to create/link to good example(s) of issues. 
We are still seeing _way_ too many issues with _**blank descriptions**_ ... 😢 

We have a _template_ in "iSearch": https://github.com/numo-labs/isearch-ui/blob/master/.github/ISSUE_TEMPLATE.md is this a _good idea_...?

Story Template see: https://en.wikipedia.org/wiki/User_story#Format

```
As a ... 
I want to ... 
so that ... 
```

**Comment by jackcarlisle** [comment]
@nelsonic I think if you are only using issues to do one job then templates are a good idea but seeing as we're using them for user stories, technical tasks and bugs, ours would have to be really general and that might defeat the purpose. Unless we had three (or however many) sections in the template and you deleted accordingly?

**Comment by nelsonic** [comment]
@jackcarlisle agree 100% (_using issues for various purposes means a single template is unlikely..._)
What we _could_ do though is make a guide similar to: https://github.com/request/request/blob/master/CONTRIBUTING.md
which shows up when you attempt to create a new issue:      https://github.com/request/request/issues/new

![request-create-issue-shows-link](https://cloud.githubusercontent.com/assets/194400/18284026/21daf7cc-7460-11e6-8d8f-00bc67afc20c.png)

**Comment by jackcarlisle** [comment]
@nelsonic yeah I like this a lot! It's a good global point of reference that we can direct people to and it's also really reusable

**Comment by jackcarlisle** [comment]
[Guideline link](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)

**Comment by nelsonic** [comment]
@jackcarlisle good link. 👍 
we are already using this in _at least one_ of our projects: https://github.com/dwyl/hapi-login/issues/new which will now point to doc you are busy updating! 🎉

**Comment by iteles** [comment]
Linked to https://github.com/dwyl/ateam-master-reference/issues/1

**Comment by nelsonic** [comment]
We _recently_ had an _example_ of a Bug reported by a Product Owner which has _specific_ detail and a checklist: https://github.com/TheScienceMuseum/collectionsonline/issues/582
This is _enough_ to get started with investigating the issue. 👍

**Comment by nelsonic** [comment]
@iteles would _this_ qualify as a good issue: https://github.com/dwyl/terminate/issues/29
and subsequent Pull Request for an Open Source Project: https://github.com/dwyl/terminate/pull/31
? 🤔  ?

**Comment by nelsonic** [comment]
Good post on the subject: http://mikeborozdin.com/post/split-user-stories-get-value-early

**Comment by nelsonic** [comment]
Alternative formulation of User Story description:
https://blog.crisp.se/2014/09/25/david-evans/as-a-i-want-so-that-considered-harmful
